src/contracts/pods/EigenPod.sol

```diff
@@ -786,6 +796,18 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         return _validatorPubkeyHashToInfo[pubkeyHash].status;
     }
 
+    function validatorPubkeyToInfo(bytes calldata validatorPubkey) external view returns (ValidatorInfo memory) {
+        require(validatorPubkey.length == 48, "EigenPod.validatorPubkeyHashToInfo must be a 48-byte BLS public key");
+
+        return _validatorPubkeyHashToInfo[_calculateValidatorPubkeyHash(validatorPubkey)];
+    }
+
+    function validatorStatus(bytes calldata validatorPubkey) external view returns (VALIDATOR_STATUS) {
```

Comment:
why not just info func

Reply:
fair, thought I might as well.  Can remove if excessive